src/openforms/js/components/formio_builder/translation.js

```diff
@@ -14,11 +14,14 @@ import jsonScriptToVar from 'utils/json-script';
 export const nlStrings = require('lang/formio/nl.json');
 export default nlStrings;
 
+let _supportedLanguages = undefined;
 export const getSupportedLanguages = () => {
-  return jsonScriptToVar('languages', {default: []});
+  if (_supportedLanguages !== undefined) return _supportedLanguages;
+  _supportedLanguages = jsonScriptToVar('languages', {default: []});
+  return _supportedLanguages;
```

Comment:
no, it's a small optimization/React principle that no direct DOM interactions should be done during React comonent lifecycles. Caching this at the module level ensures we only query the DOM once (which is relatively slow) instead of every time a component using this renders again.

I didn't profile it though.